docs/data-sources/resources.md

```diff
@@ -15,6 +15,11 @@ Resources in Twingate represent servers on the private network that clients can
 ```terraform
 data "twingate_resources" "foo" {
   name = "<your resource's name>"
+#  name_regexp = "<your resource's name>"
```

Comment:
This file should probably be removed and the one from https://github.com/Twingate/terraform-provider-twingate/pull/443/files#diff-4cf56f1941dcb68e635ce12e6ddc09fe0ca9d321a15dec471543f27ed15f4e5b should be used instead

Reply:
fixed